lib/xgu/xgux.h

```diff
@@ -0,0 +1,88 @@
+#ifndef __XGUX_H
+#define __XGUX_H
+
+#include "xgu.h"
+
+inline
+void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int count) {
+    uint32_t *p = pb_begin();
+    p = xgu_begin(p, mode);
+    p = xgu_draw_arrays(p, start, count);
+    p = xgu_end(p);
+    pb_end(p);
+}
+
+inline
+void xgux_draw_elements16(XguPrimitiveType mode, unsigned int count, const uint16_t* elements) {
```

Comment:
API is a bit confusing because this is `xgux_draw_elements..(..., count elements)` vs `xgu_element..(..., elements, count)`.

Note difference of *element* vs *elements* + order of arguments (XGU = closer to `memcpy` / XGUX closer to `glDrawElements`).

Reply:
Order of arguments fixed.
Emphasis on plural in xgux function stays for now unless you feel strongly that it is out of place.